### PR TITLE
feat: Add documentation to test file uploads

### PR DESCRIPTION
**Pull request by artandor**

### CODE REVIEW

File: core/file-upload.md [hunk 1]
```diff
@@ -297,3 +297,48 @@ uploaded cover, you can have a nice illustrated book record!
 
 Voilà! You can now send files to your API, and link them to any other resource
 in your app.
+
+## Testing
+
+To test your upload with `ApiTestCase`, you can write a method as below:
+
+```
```
Comment (alanpoulain) [inline core/file-upload.md:305]:
```suggestion
```php
```

File: core/file-upload.md [hunk 2]
```diff
@@ -297,3 +297,48 @@ uploaded cover, you can have a nice illustrated book record!
...
+        $client->request('POST', '/media_objects', [
+            'headers' => ['Content-Type' => 'multipart/form-data'],
+            'extra' => [
+                // If you have additionnal fields in your MediaObject entity, use the parameters
+                'parameters' => [
+                    'title' => 'My file uploaded'
+                ],
+                'files' => [
+                    'file' => $file
+                ]
```
Comment (alanpoulain) [inline core/file-upload.md:334]:
```suggestion
                ],
```

File: core/file-upload.md [hunk 3]
```diff
@@ -297,3 +297,48 @@ uploaded cover, you can have a nice illustrated book record!
...
+    {
+        $file = new UploadedFile('fixtures/files/image.png', 'image.png');
+        $client = self::createClient();
+
+        $client->request('POST', '/media_objects', [
+            'headers' => ['Content-Type' => 'multipart/form-data'],
+            'extra' => [
+                // If you have additionnal fields in your MediaObject entity, use the parameters
+                'parameters' => [
+                    'title' => 'My file uploaded'
```
Comment (alanpoulain) [inline core/file-upload.md:330]:
```suggestion
                    'title' => 'My file uploaded',
```

File: core/file-upload.md [hunk 4]
```diff
@@ -297,3 +297,48 @@ uploaded cover, you can have a nice illustrated book record!
...
+                ],
+                'files' => [
+                    'file' => $file
```
Comment (alanpoulain) [inline core/file-upload.md:333]:
```suggestion
                    'file' => $file,
```

**Comment by alanpoulain** [comment]
Hello, thanks for your PR, I've made minor comments but it looks good.

**Comment by artandor** [comment]
> Hello, thanks for your PR, I've made minor comments but it looks good.

Thanks ! I wasn't sure wether or not i should target main branch or current release. I got here by clicking the link at the bottom page of the documentation.

Should i remove the "parameters" array with custom fields ? It won't work out of the box with the code above, but on the other hand i struggled a bit to find that i shouldn't put this in body but in parameters instead.

File: core/file-upload.md [hunk 5]
```diff
@@ -297,3 +297,48 @@ uploaded cover, you can have a nice illustrated book record!
...
+
+    public function testCreateAMediaObject(): void
+    {
+        $file = new UploadedFile('fixtures/files/image.png', 'image.png');
+        $client = self::createClient();
+
+        $client->request('POST', '/media_objects', [
+            'headers' => ['Content-Type' => 'multipart/form-data'],
+            'extra' => [
+                // If you have additionnal fields in your MediaObject entity, use the parameters
```
Comment (alanpoulain) [inline core/file-upload.md:328]:
```suggestion
                // If you have additional fields in your MediaObject entity, use the parameters.
```

**Comment by alanpoulain** [comment]
Targeting `2.6` is fine. You can leave the parameters, I think the comment is clear enough.

**Review by alanpoulain — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thank you!

**Pull request merged**